frontend/src/features/entities/EntityList.tsx

```diff
@@ -14,6 +15,26 @@ interface Props {
   personalScores?: { [uid: string]: number };
 }
 
+const AvailableEntity = ({
+  children,
+  uid,
+}: {
+  children: React.ReactNode;
+  uid: string;
+}) => {
+  const [isAvailableOnYoutube, setIsAvailableOnYoutube] = useState(false);
```

Comment:
The `EntityList` can be used with any `entity.type`. So this component should not assume the entity is a Youtube video.